README.md

```diff
@@ -209,8 +217,28 @@ The device or market can unselect the Aggregator:
 device.unselect_aggregator(aggregator.aggregator_uuid)
 ```
 
+#### Available Aggregator's methods
+
+`Aggregator` instances provide methods that can simplify specific operations. Below we list some of the most commonly used:
+
+- Return all the aggregators connected to the aggregator's simulation: 
+    ```python
+    list_aggregators()
```

Comment:
As a side comment, the list-aggregatorrs should not reside in an aggregator instance, it should be a standalone function

Reply:
I agree. I suggest we create a task for this.